### ISSUE TITLE
[website] toggle button uses color alone to communicate state

### ISSUE THREAD

**Issue by asadotzler** [mozilla/fx-private-relay#227]
The toggle button at https://relay.firefox.com/accounts/profile/ uses a blue-green gradient border and green dot to communicate that it's on and a red-purple gradient border and red dot to communicate that it's off. This is insufficient for accessibility as some color blind people may not be able to distinguish the states.

**Comment by groovecoder** [comment]
iOS puts a "1" and "0" in the toggle to indicate on/off, but I'm not sure that's universally understood?

Maybe a ✅  or a ❌ in the toggle?

**Comment by hafta** [comment]
<img width="66" alt="Screen Shot 2020-04-29 at 3 02 56 PM" src="https://user-images.githubusercontent.com/10162738/80651324-9584f680-8a2a-11ea-94a4-23c048ca31ee.png">

<img width="75" alt="Toggle" src="https://user-images.githubusercontent.com/10162738/80651248-6ff7ed00-8a2a-11ea-801d-255bb8ba6478.png">

To my eyes, when toggled to the right (enabled state), the toggle appears faded and therefore I incorrectly assumed that was the disabled state. I think it would be more clear if the colors were more bold when the toggle is enabled and more muted/faded colors (or gray) when the toggle is disabled.

Additionally, I would prefer to have some text indicating the forwarding is disabled. It would be redundant, but might help with clarity. Something like `Forward emails to foo@mozilla.com. Disabled.`

And, is it redundant to include the forwarding destination address for each forward? Considering that all emails will forward to the same Firefox Account address.

Can the toggle have alt text indicating current state?
